examples/gnrc_border_router/README.md

```diff
@@ -1,4 +1,144 @@
-# gnrc_networking_border_router example
+# gnrc_border_router using automatic configuration
+A new setup is possible on which only one serial interface is needed. 
```

Comment:
Since this is now the default config, how about rephrasing the first two sentences with something like "This setup uses a single serial interface, ethos (Ethernet Over Serial) and UHCP (micro Host Control Protocol).